# 3.5 Literature

## Predict structure probing data

* Delli Ponti, R., Marti, S., Armaos, A., and Tartaglia, G.G. (2016). A high-throughput approach to profile RNA structure. Nucl. Acids Res. gkw1094.

## Structure probing

* **icSHAPE (mouse)**: Spitale, R.C., Flynn, R.A., Zhang, Q.C., Crisalli, P., Lee, B., Jung, J.-W., Kuchelmeister, H.Y., Batista, P.J., Torre, E.A., Kool, E.T., et al. (2015). Structural imprints in vivo decode RNA regulatory mechanisms. Nature 519, 486–490.
* **icSHAPE and PARIS (human)**: Lu, Z., Zhang, Q.C., Lee, B., Flynn, R.A., Smith, M.A., Robinson, J.T., Davidovich, C., Gooding, A.R., Goodrich, K.J., Mattick, J.S., et al. (2016). RNA Duplex Map in Living Cells Reveals Higher-Order Transcriptome Structure. Cell 165, 1267–1279.
* **COMARDES**: Ziv, O., Gabryelska, M.M., Lun, A.T.L., Gebert, L.F.R., Sheu-Gruttadauria, J., Meredith, L.W., Liu, Z.-Y., Kwok, C.K., Qin, C.-F., MacRae, I.J., et al. (2018). COMRADES determines in vivo RNA structures and interactions. Nature Methods 1.
* **SHAPE-MaP (E. coli)**: Mustoe, A.M., Busan, S., Rice, G.M., Hajdin, C.E., Peterson, B.K., Ruda, V.M., Kubica, N., Nutiu, R., Baryza, J.L., and Weeks, K.M. (2018). Pervasive Regulatory Functions of mRNA Structure Revealed by High-Resolution SHAPE Probing. Cell 173, 181-195.e18.
* **SHAPE-MaP (canonical ncRNA)**: Siegfried, N.A., Busan, S., Rice, G.M., Nelson, J.A.E., and Weeks, K.M. (2014). RNA motif discovery by SHAPE and mutational profiling (SHAPE-MaP). Nat Meth 11, 959–965.
* **DMS-seq (human)**: Rouskin, S., Zubradt, M., Washietl, S., Kellis, M., and Weissman, J.S. (2014). Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo. Nature 505, 701–705.
* **PARS (human)**: Wan, Y., Qu, K., Zhang, Q.C., Flynn, R.A., Manor, O., Ouyang, Z., Zhang, J., Spitale, R.C., Snyder, M.P., Segal, E., et al. (2014). Landscape and variation of RNA secondary structure across the human transcriptome. Nature 505, 706–709.
* **Mutate-and-map (canonical ncRNA)**: Kladwang, W., VanLang, C.C., Cordero, P., and Das, R. (2011). A two-dimensional mutate-and-map strategy for non-coding RNA structure. Nat Chem 3, 954–962.
* **DMS and translation (E. coli)**: Zhang, Y., Burkhardt, D.H., Rouskin, S., Li, G.-W., Weissman, J.S., and Gross, C.A. (2018). A Stress Response that Monitors and Regulates mRNA Structure Is Central to Cold Shock Adaptation. Molecular Cell 70, 274-286.e7.
* **DMS-MaPseq (yeast and human)**: Zubradt, M., Gupta, P., Persad, S., Lambowitz, A.M., Weissman, J.S., and Rouskin, S. (2017). DMS-MaPseq for genome-wide or targeted RNA structure probing in vivo. Nature Methods 14, 75–82.
* **Structure probing in vivo (Review)**: Spitale, R.C., Crisalli, P., Flynn, R.A., Torre, E.A., Kool, E.T., and Chang, H.Y. (2013). RNA SHAPE analysis in living cells. Nat Chem Biol 9, 18–20.
* **Structure probing and RNA modification (Review)**: Incarnato, D., and Oliviero, S. (2017). The RNA Epistructurome: Uncovering RNA Function by Studying Structure and Post-Transcriptional Modifications. Trends in Biotechnology 35, 318–333.
* **RNAex (database)**: Wu, Y., Qu, R., Huang, Y., Shi, B., Liu, M., Li, Y., and Lu, Z.J. (2016). RNAex: an RNA secondary structure prediction server enhanced by high-throughput structure-probing data. Nucleic Acids Res 44, W294–W301.
* **RISE (database)**: Gong, J., Shao, D., Xu, K., Lu, Z., Lu, Z.J., Yang, Y.T., and Zhang, Q.C. RISE: a database of RNA interactome from sequencing experiments. Nucleic Acids Res.

## Analysis

* **Differential SHAPE**: Choudhary, K., Lai, Y.-H., Tran, E.J., and Aviran, S. (2019). dStruct: identifying differentially reactive regions from RNA structurome profiling data. Genome Biology 20, 40.
* **Alternative structure**: Li, H., and Aviran, S. (2018). Statistical modeling of RNA structure profiling experiments enables parsimonious reconstruction of structure landscapes. Nature Communications 9, 606.
* **Inference from structure probing data**: Selega, A., Sirocchi, C., Iosub, I., Granneman, S., and Sanguinetti, G. (2017). Robust statistical modeling improves sensitivity of high-throughput RNA structure probing experiments. Nat Meth 14, 83–89.


## RNA structure motif discovery

* **Motif search in structure probing data**: Ledda, M., and Aviran, S. (2018). PATTERNA: transcriptome-wide search for functional RNA elements via structural data signatures. Genome Biology 19, 28.
* **patteRNA improved version**: Radecki, P., Ledda, M., and Aviran, S. (2018). Automated Recognition of RNA Structure Motifs by Their SHAPE Data Signatures. Genes 9, 300.
* **Structure alignment**: Smith, M.A., Seemann, S.E., Quek, X.C., and Mattick, J.S. (2017). DotAligner: identification and clustering of RNA structure motifs. Genome Biology 18, 244.
* **Motif discovery for RBP (Review)**: Sasse, A., Laverty, K.U., Hughes, T.R., and Morris, Q.D. (2018). Motif models for RNA-binding proteins. Current Opinion in Structural Biology 53, 115–123.
* **Motif discovery (Review)**: Achar, A., and Sætrom, P. (2015). RNA motif discovery: a computational overview. Biology Direct 10, 61.

## Deep learning for motif discovery

* **RNA-protein, DNA-protein (DeepBind)**: Alipanahi, B., Delong, A., Weirauch, M.T., and Frey, B.J. (2015). Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning. Nat Biotech advance online publication.
* **DNA accessibility (Basset)**: Kelley, D.R., Snoek, J., and Rinn, J.L. (2016). Basset: learning the regulatory code of the accessible genome with deep convolutional neural networks. Genome Res.
* **RNA-protein (iDeep)**: Pan, X., and Shen, H.-B. (2017). RNA-protein binding motifs mining with a new hybrid deep learning based cross-domain knowledge integration approach. BMC Bioinformatics 18, 136.
* **Regulatory activity**: Kelley, D.R., Reshef, Y.A., Bileschi, M., Belanger, D., McLean, C.Y., and Snoek, J. (2018). Sequential regulatory activity prediction across chromosomes with convolutional neural networks. Genome Res. 28, 739–750.









